api/mfa.go

```diff
@@ -51,3 +58,45 @@ func (a *API) DisableMFA(w http.ResponseWriter, r *http.Request) error {
 	}
 	return sendJSON(w, http.StatusOK, user)
 }
+
+func (a *API) GenerateRecoveryCodes(w http.ResponseWriter, r *http.Request) error {
+	const NUM_RECOVERY_CODES = 8
+	const RECOVERY_CODE_LENGTH = 8
```

Comment:
nit: as much as i hate to say this, the standard convention in go is to use camelCase and the first letter is either an uppercase or lowercase depending on whether you want it to be exported or not.